mani_skill2/utils/dataset_download.py

```diff
@@ -0,0 +1,54 @@
+import os.path as osp
```

Comment:
Can you add a main function so that users can download from the command line as well